src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-from distutils.version import StrictVersion  # pylint: disable=deprecated-module
+from packaging.version import Version
```

Comment:
Ref: https://github.com/pypa/packaging/issues/520

Reply:
Core migrated to `packaging.version.parse` long ago: https://github.com/Azure/azure-cli/pull/17667.

`packaging.version.parse` is the same as `packaging.version.Version`: https://github.com/pypa/packaging/blob/bb61b7bcbc429b49e6659a8b6c2966b6ebd046b6/src/packaging/version.py#L47-L56